grudge/discretization.py

```diff
@@ -304,33 +313,32 @@ def connection_from_dds(self, from_dd, to_dd):
         else:
             raise ValueError("cannot interpolate from: " + str(from_dd))
 
-    def group_factory_for_quadrature_tag(self, quadrature_tag):
+    def group_factory_for_quadrature_tag(self, discretization_tag):
```

Comment:
Rename/deprecate this method?

Reply:
@inducer Sure. Do you want to leave `quad_tag_to_group_factory` argument alone in the class constructor? or should we change that also (to `discr_tag_to_group_factory`)?

Reply:
I'd say change it, too. I think we're phasing out the word "quadrature tag", so this makes sense to me.

Reply:
This function was renamed in this commit: https://github.com/inducer/grudge/pull/83/commits/6c8c3d94fdccd47f6fad605866ff1f37a5cae167, and the `quad_tag_to_group_factory` argument was deprecated in these commits (which includes references to them within the test suite/examples): https://github.com/inducer/grudge/pull/83/commits/fdafb6dbccf084082c076e0f9ad1ef2d12ee71d8, https://github.com/inducer/grudge/pull/83/commits/f239490d1045865f0147cde6b64f2540591043e4